src/main/java/com/attorneyrequest/validation/ValidationWorkflow.java

```diff
@@ -0,0 +1,24 @@
+package com.attorneyrequest.validation;
+
+import com.attorneyrequest.common.Case;
+import com.attorneyrequest.data_entry.DataEntryWorkflow;
+
+public class ValidationWorkflow {
+    private static Case caseFile = null;
+    private static DataEntryWorkflow dataFlow = new DataEntryWorkflow();
+
+    public ValidationWorkflow()
+    {
+        caseFile = new Case();

```

Comment:
Validation workflow shouldn't create another caseFile object, this object is already created from Data entry. Use the one that pass from Data entry instead. It will be in DataEntry_Workflow class. I will just merge this for now.